cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -677,7 +677,13 @@ set(Python3_FIND_FRAMEWORK LAST)
 # Even if we don't build PyROOT, one still need python executable to run some scripts
 list(APPEND python_components Interpreter)
 if(pyroot OR tmva-pymva)
-  list(APPEND python_components Development)
+  if(DEFINED ROOT_WHEEL_BUILD AND ROOT_WHEEL_BUILD)
```

Comment:
If I understand correctly, we only need the full `Development` component when building `tpython` or `tmva-pymva`.

Maybe this logic works then in general?

```python
if(tpython OR tmva-pymva)
  list(APPEND python_components Development)
elseif(pyroot)
  list(APPEND python_components Development.Module)
endif()
```

Then there is no need for a code branch specific to `ROOT_WHEEL_BUILD`.